gomod2nix.toml

```diff
@@ -3602,12 +3602,13 @@
     sha256 = "11b4vjldnp6ws52wlqrzsy6nnn5qa2m98wiqvd90nzsh0n3bi9nz"
 
 ["github.com/tecbot/gorocksdb"]
```

Comment:
should be `github.com/cosmos/gorocksdb`

Reply:
this file is generated automatically by tool `gomod2nix`, so it's fine.